object.go

```diff
@@ -457,7 +466,7 @@ func (d *Decoder) decInstance(typ reflect.Type, cls classInfo) (interface{}, err
 			}
 
 		default:
-			return nil, perrors.Errorf("unknown struct member type: %v %v", kind, typ.Name()+"."+typ.Field(index).Name)
+			return nil, perrors.Errorf("unknown struct member type: %v %v", kind, typ.Name()+"."+typ.FieldByIndex(index).Name)
```

Comment:
Is it necessary to return error ?

for example, the server api upgrade and add a new field, the client code would upgrade later. between this time, the client will don't work

Reply:
another case, if json bytes has more fields, json.unmarshal will ignore the unused fields and don't report any error.